### PR TITLE
Xiaoyu

### PR DESCRIPTION
**Pull request by ZhangHuafan**
Okayyy!

### CODE REVIEW

**Comment by le0tan** [comment]
@ZhangHuafan you can just ignore the fancy stacked card view that I left for the first page... It was, unfortunately, another piece of my experiment... Just display the announcements as normal card views should do =)

File: lib/card_scroll_widget.dart [hunk 1]
```diff
@@ -9,11 +10,16 @@ class CardScrollWidget extends StatelessWidget {
...
   final double padding = 20.0;
   final double verticalInset = 20.0;
+ 
+  
 
   CardScrollWidget(this.currentPage);
-
   @override
   Widget build(BuildContext context) {
+    //写得不太好这种方法虽然方便
```
Comment (le0tan) [inline lib/card_scroll_widget.dart]: Please use English all the time for commits.

**Review by le0tan — CHANGES_REQUESTED** [review]
There's no need to use the stacked card view for announcements, please change announcements to a simple list view similar to Twitter feeds.

File: lib/card_scroll_widget.dart (hunk 1 above)
Comment (ZhangHuafan) [reply lib/card_scroll_widget.dart]: Okay Okay!

**Review by le0tan — APPROVED** [review]

**Pull request merged**